docs/reference/organizations-on-nuget-org.md

```diff
@@ -0,0 +1,105 @@
+
+![Add new organization page on nuget.org](media/org-add-new-page.png)
+
+Once the organization account is created, you are the administrator and can submit packages for the organization and add organization members.
+
+### Transform existing account to an organization
+
+If you're managing packages as a team using a single user account and would like to convert that account into an organization, use the **Transform your account to an organization** option on the **Manage Organizations** page:
+
+![Option on nuget.org to transform an existing account to an organization](media/org-transform-option.png)
+
+On the next page, specify different user account to assign as the administrator of the organization, then select **Transform**.
+
+![Entering information for transforming a user account to an organization](media/org-transform-page.png)
+
+> [!Warning]
+> Account conversion is irreversible: you cannot transform an organization back to a user account.
+
+## Managing organization members
+
+As the organization administrator, you can add members by providing each member's nuget.org *user account name*; email addresses cannot be used. You then mark each member as a collaborator or administrator with the following permissions:
+
+| Permission | Collaborator | Administrator |
+| --- | --- | --- |
+| Manage the organization's packages<br/>(submit new packages, update or unlist existing packages) | Yes | Yes |
+| Change organization metadata<br/>(email address, notification settings) | No | Yes |
+| Manage organization members | No | Yes |
+| Request or act on co-ownership requests for organization packages | No | Yes |
+
+## Managing packages
+
+As an organization member, you can see all the organization's packages in the **Packages** section of the organization's details page. On the **Manage Packages** page you can also view packages across your account and all organizations of which you're a member. You can select the account filter to view packages by your account or any specific organization.
+
+![Managing packages with the account filter](media/org-manage-packages-option.png)
+
+## Publishing packages
+
+You publish packages to an organization like you publish packages to a user account, by directly uploading the package to nuget.org or by pushing the package through the `nuget push` or `dotnet nuget push` CLI commands.
+
+### Uploading packages
+
+When you directly upload a new package on the [nuget.org Upload](https://www.nuget.org/packages/manage/upload) page, you assign the package owner to a user or organization account :
+
+![Upload package with account option](media/org-upload-option.png)
+
+When submitting updates to an existing package, you don't see the account dropdown because package ownership is already known to nuget.org.
```

Comment:
```
The owner dropdown does not appear when uploading new versions of an existing package because the package's ownership is already known to nuget.org.
```

(`you don't see the account dropdown` is a really awkward phrase)

Reply:
Consider: "For exisitng packages, owners are pre-determined, and the `account dropdown` will not appear."

Reply:
I think this clarification is important to prevent confusion. Users might not realize they're pushing an update to a package and get curious why they can't specify the owner of the package.